central/processlisteningonport/datastore/datastore.go

```diff
@@ -0,0 +1,36 @@
+package datastore
+
+import (
+	"context"
+
+	processIndicatorStore "github.com/stackrox/rox/central/processindicator/datastore"
+	"github.com/stackrox/rox/central/processlisteningonport/store/postgres"
+	"github.com/stackrox/rox/generated/storage"
+)
+
+// WalkFn is a convenient type alias to use for the Walk function
+type WalkFn = func(plop *storage.ProcessListeningOnPortStorage) error
+
+// DataStore interface for ProcessListeningOnPort object interaction with the database
+//
+//go:generate mockgen-wrapper
+type DataStore interface {
```

Comment:
nit:  We should probably be consistent at least for the exported interface routines.
`AddProcessListeningOnPort`
`GetProcessListeningOnPort`
`RemovePLOP`

IMO we should either shorten them all or spell them all out.

Reply:
RemovePLOP was renamed to RemoveProcessListeningOnPort.